konfera/templates/konfera/event/base.html

```diff
@@ -0,0 +1,42 @@
+{% extends 'konfera/base.html' %}
+{% load i18n %}
+
+{% block title %}
+    {{ event }}
+{% endblock %}
+
+{% block content %}
+<div class="row">
+  <div class="col-md-{% if sponsors %}8{% else %}12{% endif %}">
+    {% block event_details %}{% endblock %}
+  </div>
+
+  {% if sponsors %}
+  <div class="col-md-4">
+  <!-- right_nav -->
+    <div class="text-center right-nav hidden-print">
+      {% for sponsor in sponsors %}
+        {% ifchanged sponsor.type %}
+          {% if not forloop.first %}
+      </ul>
```

Comment:
I think it would make sense (for readability) to move this `</ul/<br><hr>` to the end of forloop next to `forloop.last`. I took me a minute to realize what is happening.